srcpkgs/musl-legacy-compat/files/error.h

```diff
@@ -26,4 +26,31 @@ static inline void error(int status, int errnum, const char* format, ...)
 		exit(status);
 }
 
+static int error_one_per_line = 0;
+
+static inline void error_at_line(int status, int errnum, const char *filename,
+		unsigned int linenum, const char *format, ...)
+{
+	va_list ap;
+	if (error_one_per_line) {
+		static const char *old_filename;
```

Comment:
plocate doesn't try to use this from different TUs, does it? Otherwise it might have some weird results.

Reply:
Yea it doesn't, but glibc does the same so this shouldn't be any worse.

Reply:
LGTM, then.

Do you think we could extend the warning in the header to mention that this can be a flawed workaround, depending on how upstream uses the functions?

Reply:
hm yea not sure if its worth it, after looking at the actual implementation of this in glibc, its probably better to just patch the programs using it and get it upstreamed, there is no real value in `error.h`.
I was expecting there to be some localization stuff or similar.